apps/hash-frontend/src/pages/[shortname].page/profile-page-tabs.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Skeleton } from "@mui/material";
+import { FunctionComponent } from "react";
+
+import { Org, User } from "../../lib/user-and-org";
+import { TabLink } from "../../shared/ui/tab-link";
+import { Tabs } from "../../shared/ui/tabs";
+import { ProfilePageTab } from "./util";
+
+export const ProfilePageTabs: FunctionComponent<{
+  profile?: User | Org;
+  tabs: ProfilePageTab[];
+  currentTab: ProfilePageTab;
+}> = ({ tabs, currentTab, profile }) => {
+  const currentTabLabel =
+    currentTab.kind === "profile" ? currentTab.title : currentTab.pluralTitle;
+
+  return (
+    <Box sx={{ overflowX: "scroll" }}>
```

Comment:
We could made this `auto` so that the scrollbar isn't introduced unless necessary. There will only be layout shift if someone is resizing the window and it appears. We probably also want to style the scrollbar to make it less intrusive.

I also get a vertical scrollbar
![Screenshot 2023-10-10 at 12 22 10](https://github.com/hashintel/hash/assets/37743469/6e568205-d40f-4fde-9645-9b06af0f319d)

Reply:
I've made the overflow `auto` in https://github.com/hashintel/hash/pull/3338/commits/e3f3674a4d2fdd8ee782d0de480840fd923e454d, I can't reproduce the vertical scrollbar. I don't really get why chrome sometimes adds these intrusive scrollbars, and other times uses the native overlaying ones. I'm not sure its worth modifying the styling of this further as I need to rework this once I implement [the updated designs](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH?node-id=9717%3A50623&mode=dev) anyway as part of https://linear.app/hash/issue/H-931/display-cover-image-in-user-profile-page

Reply:
We spoke about this and figured out it was because you have overlay scrollbars in Chrome and I have the solid gray ones (that shift the layout). Applying `overflow-y: hidden` to the tabs (somewhat hackily) fixes it